### PR TITLE
Fix for error 404 in documentation

### PR DESCRIPTION
**Pull request by swapsha96**
Addressing to issue #2169: The change made in this PR stops Sphinx from adding edit button on generated docs, resolving this issue. I went through [AstroPy conf.py](https://github.com/astropy/astropy/blob/master/docs/conf.py#L203) to understand the problem so I cannot test it since I don't know anything about Sphinx. Please, if someone checks this, I would be grateful.

### CODE REVIEW

**Comment by Cadair** [comment]
You can build the documentation locally by running `python setup.py build_docs -o` to test.

**Comment by swapsha96** [comment]
Thanks, @Cadair! But I've no idea why I am getting this after running the command? :-/
screenshot from 2017-07-03 20-50-45

**Comment by swapsha96** [comment]
@Cadair @dpshelio Checked and it works.

File: doc/source/conf.py [hunk 1]
```diff
@@ -162,6 +162,8 @@
 edit_on_github_source_root = ""
 edit_on_github_doc_root = "docs"
 
+edit_on_github_skip_regex = '_.*|api/.*'
```
Comment (Cadair) [inline doc/source/conf.py]: I think this will work better if it's `generated` rather than `api`? Have you tested to see if this works as is?

**Pull request closed without merging**